tests/inte_test/test_dssm.py

```diff
@@ -26,11 +30,30 @@ def inte_test_dssm():
     # do pre-processing.
     dssm_preprocessor = preprocessor.DSSMPreprocessor()
```

Comment:
Here, we also need to save `dssm_preprocessor ` for prediction phase. Thus, we can load `dssm_preprocessor` for testing.